kubeflow/compiled/train_on_synthetic_and_real_dataset.yaml

```diff
@@ -2,12 +2,12 @@ apiVersion: argoproj.io/v1alpha1
 kind: Workflow
 metadata:
   generateName: train-on-synthetic-real-world-dataset-
-  annotations: {pipelines.kubeflow.org/kfp_sdk_version: 1.0.0, pipelines.kubeflow.org/pipeline_compilation_time: '2020-08-27T18:59:05.348860',
+  annotations: {pipelines.kubeflow.org/kfp_sdk_version: 1.0.0, pipelines.kubeflow.org/pipeline_compilation_time: '2020-08-28T12:34:16.260422',
     pipelines.kubeflow.org/pipeline_spec: '{"description": "Train on Synthetic + Real
       World Dataset", "inputs": [{"default": "unitytechnologies/datasetinsights:latest",
       "name": "docker", "optional": true, "type": "String"}, {"default": "https://storage.googleapis.com/datasetinsights/data/groceries/v3.zip",
       "name": "source_uri", "optional": true, "type": "String"}, {"default": "datasetinsights/configs/faster_rcnn_fine_tune.yaml",
-      "name": "config", "optional": true, "type": "String"}, {"default": "https://storage.googleapis.com/datasetinsights/models/sim2real/FasterRCNN.estimator",
+      "name": "config", "optional": true, "type": "String"}, {"default": "https://storage.googleapis.com/datasetinsights/models/Synthetic/FasterRCNN.estimator",
```

Comment:
Current directory [structure](https://console.cloud.google.com/storage/browser/datasetinsights/models;tab=objects?authuser=1&project=unity-ai-thea-test&prefix=) is different.